packages/wsr-types/__tests__/Selector-tests.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import Selector from 'wix-style-react/Selector';
+import { selectorTestkitFactory } from 'wix-style-react/dist/testkit';
+import { selectorTestkitFactory as selectorEnzymeTestkitFactory } from 'wix-style-react/dist/testkit/enzyme';
+import * as enzyme from 'enzyme';
+
+function SectionHelperWithMandatoryProps() {
```

Comment:
SelectorWithMandatoryProps
```suggestion
function SectionHelperWithMandatoryProps() {
```